### DISCUSSION TITLE
How to unselect all options filtered by search?

### DISCUSSION THREAD

**Discussion by jelly-garden** [sa-si-dev/virtual-select#116] [Q&A]
If I set `selectAllOnlyVisible`, I can select only visible options on clicking select all checkbox when options filtered by search.
But I cannot find how to run reverse.
Is there any way to unselect all options after clicking select all checkbox when options filtered by search?

**Comment by sa-si-dev** [comment]
As of now there is no feature like this. I will implement it in the future release.
